WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -445,6 +445,16 @@ class AztecPostViewController: UIViewController, PostEditor {
         return AztecVerificationPromptHelper(account: self.post.blog.account)
     }()
 
+    /// The view to show when media picker has no assets to show.
+    ///
+    fileprivate let noResultsView: WPNoResultsView = {
+        let noResultsView = WPNoResultsView()
+        noResultsView.accessoryView = UIImageView(image: UIImage(named: "media-no-results"))
```

Comment:
This configuration can probably be added to the Media extension  for WPNoResultsView has a convenience init. no?

Reply:
Good point!